contracts/deploy/00_deploy_DC.ts

```diff
@@ -0,0 +1,50 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { ethers } from 'hardhat'
+import { DeployFunction } from 'hardhat-deploy/types'
+import config from '../config'
+import fs from 'fs/promises'
+import { chunk, max } from 'lodash'
+import { deflateSync } from 'zlib'
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
the migration and initialization code is missing. Not needed at the moment, but will be useful later